src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -556,30 +600,45 @@ handle_message({neighbor_accepted, Peer, Tag, Sender},
     {reply, ok, State};
 
 handle_message({neighbor_rejected, Peer, _Sender}, State) ->
+    %% @todo Is this necessary?
```

Comment:
As I know, the `gen_server:cast(?MODULE, {disconnect, Peer})` will close the open connection.